clouddriver-ecs/src/test/groovy/com/netflix/spinnaker/clouddriver/ecs/deploy/ops/CreateServerGroupAtomicOperationSpec.groovy

```diff
@@ -73,6 +73,15 @@ class CreateServerGroupAtomicOperationSpec extends CommonAtomicOperation {
 
   def service = new Service(serviceName: "${serviceName}-v008")
 
+  def commonClientInitialization() {
```

Comment:
If you call this method "setup", it will automatically run prior to each test

Example:
https://github.com/spinnaker/clouddriver/blob/master/clouddriver-ecs/src/test/groovy/com/netflix/spinnaker/clouddriver/ecs/provider/agent/TaskHealthCachingAgentSpec.groovy#L62-L63

Reply:
I didn't want to run it for every test though, just specific cases, do you think it's better to define it as setup and then override it in all the other test cases, or as a method and call it in the tests that need it?

Reply:
I don't think any of the other tests require different setup - the two existing ones that do this share the same setup, and the rest call methods that don't use the client provider.

Reply:
Updated